datalad/customremotes/ria_utils.py

```diff
@@ -99,7 +101,16 @@ def verify_ria_url(url, cfg):
                          "Supported: ssh, file, http(s)" %
                          protocol)
 
-    return url_ri.hostname if protocol != 'file' else None, \
+    host = '{user}{pdelim}{passwd}{udelim}{hostname}{portdelim}{port}'.format(
+        user=url_ri.username or '',
+        pdelim=':' if url_ri.password else '',
+        passwd=url_ri.password or '',
+        udelim='@' if url_ri.username else '',
+        hostname=url_ri.hostname or '',
+        portdelim=':' if url_ri.port else '',
+        port=url_ri.port or '',
+    )
```

Comment:
not familiar with RIA enough and can't grasp from the docstring -- what is the purpose/use of this returned entry?
it seems no longer a URL (no `protocol://` prefix) and also might contain sensitive information (`password`) thus should probably not be logged anywhere  (although I just now realized that we might be logging such URLs left and right) and should not be served as some kind of a "annex remote description".

is that a `host` to ssh into? (I guess not since IIRC you can't even add `:port` not to say password like that)

Reply:
> is that a host to ssh into?

If it's SSH, then yes. Will be passed to our SSH machinery in that case. 

Overall it depends: It's the thing the `base_path` of the RIA store is referring to.

Reply:
> I guess not since IIRC you can't even add :port not to say password like that

Thx @yarikoptic for making this point. I think it is better to switch to URL style destination description right away to actually make this possible. I will push an update.